scripts/download/tools.mjs

```diff
@@ -135,6 +135,45 @@ export default async function main(platform) {
     throw new Error(`Matched ${ kimSHA.length } hits, not exactly 1, for platform ${ kubePlatform } in [${ allKimSHAs }]`);
   }
   await download(kimURL, kimPath, { expectedChecksum: kimSHA[0].split(/\s+/, 1)[0] });
+
+  // Download Trivy
+  // Always run this in the VM, so download the *LINUX* version into binDir
```

Comment:
Why move it into its own file, when what it's doing is similar to the other tools (except hardwiring the platform to linux)?

Reply:
I guess it would make more sense to say, would it make sense to split all Linux-based tools to their own file (currently it would be just this one), so we can do it without any of the platform logic in the file?

Reply:
I don't see how this reduces complexity that much (which is sort of what this kind of thing is all about). Every tool has its own quirks, but there's enough of a common pattern that it's easier to understand what `tools.mjs` does by having everything in the one file.